databricks_cli/runs/cli.py

```diff
@@ -113,13 +113,19 @@ def _runs_to_table(runs_json):
 @click.option('--limit', default=None, type=int,
               help='The limit determines the number of runs listed. '
                    'Limit must be between 0 and 1000. Set to 20 runs by default.')
+@click.option('--expand-tasks', is_flag=True,
+              help='Expands the tasks array (only available in API 2.1).')
+@click.option('--all', '_all', is_flag=True,
```

Comment:
This is potentially a problem because runs/list is essentially unbounded. So this would result in nearly infinite requests or a DOS